xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/CheckCash.java

```diff
@@ -54,18 +54,18 @@ static ImmutableCheckCash.Builder builder() {
   }
 
   /**
-   * Set of {@link TransactionFlags}s for this {@link AccountDelete}, which only allows the
-   * {@code tfFullyCanonicalSig} flag.
+   * Set of {@link TransactionFlags}s for this {@link CheckCash}, which only allows the
+   * {@code tfFullyCanonicalSig} flag, which is deprecated.
    *
    * <p>The value of the flags cannot be set manually, but exists for JSON serialization/deserialization only and for
    * proper signature computation in rippled.
    *
-   * @return Always {@link TransactionFlags} with {@code tfFullyCanonicalSig} set.
+   * @return Always {@link TransactionFlags#EMPTY}.
    */
   @JsonProperty("Flags")
   @Value.Derived
```

Comment:
New path forward:

1. For all `Transactions` change all flags' `@Derived` method to be `@Default`.
2. Introduce the following into `TransactionFlags`: 
  
`public static final TransactionFlags UNSET = new TransactionFlags(0);`

*Ramifications*

1. New code will work fine.
2. No compilation build will fail with the change.
3. Old code will work fine, because all existing builder code will still work (in a move-forward capacity).
4. The default value of newly created transactions will be `EMPTY`, which would only affect (maybe) a unit test in a project used by the library, but in this case such a unit test is incorrect (i.e., a bug)